reporter/src/main/kotlin/utils/FreemarkerTemplateProcessor.kt

```diff
@@ -120,7 +121,8 @@ class FreemarkerTemplateProcessor(
             "licenseTextProvider" to input.licenseTextProvider,
             "LicenseView" to LicenseView,
             "helper" to TemplateHelper(input.ortResult, input.licenseClassifications, input.resolutionProvider),
-            "projectsAsPackages" to projectsAsPackages
+            "projectsAsPackages" to projectsAsPackages,
+            "vulnerabilityReference" to VulnerabilityReference
```

Comment:
I'd prefer the changes to this file to be done only when the templates actually start using it.

Reply:
So custom templates depends on the default template?

Reply:
Ah, you mean that we should *provide* the feature even if it's not (yet) *used* in the default template. Yes, I'd agree to that. But then again please amend you existing commit message with something like "Also make the method available to all templates by default."

Reply:
> The method is available to all templates by default, because it's already added to the tempalte data model.

I'm perfectly fine with not using my suggested wording, but this sentence read odd to me (besides the typo in "tempalte"). How about: "The method is made available to all templates by default by adding to the template data model."